conans/test/functional/generators/cmake_find_package_test.py

```diff
@@ -13,6 +13,110 @@
 from conans.test.utils.tools import TestClient, NO_SETTINGS_PACKAGE_ID
 
 
+@pytest.mark.slow
+@pytest.mark.tool_cmake
+class TestCMakeFindPackageGenerator:
+
+    @pytest.mark.parametrize("use_components", [False, True])
+    def test_build_modules_alias_target(self, use_components):
```

Comment:
If this is for injecting a build_module for a dependency, could we better test it without actually needing to build any binaries? I think we need to reduce the tests a little bit, specially if we can avoid compiling that would be great.

Reply:
done